gplog/gplog.go

```diff
@@ -59,6 +60,16 @@ const (
 	LOGDEBUG
 )
 
+// --- constants for colorized output support
+const escape = "\x1b"
```

Comment:
```suggestion
const ESCAPE = "\x1b"
```
Can we capitalize it such that we can denote it's a const?